mtprotoproxy.py

```diff
@@ -133,6 +146,9 @@ def decrypt(self, data):
 my_ip_info = {"ipv4": None, "ipv6": None}
 
 
+FLAGHACKS = {}
```

Comment:
There's got to be a better way to store per-connection info than this. If there isn't, this needs to be cleaned up after disconnects